### ISSUE TITLE
[Accessibility] Text has insufficient contrast

### ISSUE THREAD

**Issue by gazjoy** [Adyen/adyen-web#1800]
**Describe the bug**
When reviewing our checkout for [WCAG 2.1 - 1.4.3 Contrast (minimum) (AA)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html), I found the payment step to not meet the minimum contrast ratio.

**To Reproduce**
1. Go to 'payment step'
2. View payment step fields placeholder text
3. See insufficient contrast
4. Enter data
5. See insufficient contrast (green text)

**Expected behavior**
Text to meet minimum contrast ratio 3.5:1.

**Screenshots**
![image](https://user-images.githubusercontent.com/628064/196161195-a65e111a-edc1-4300-a768-e77f2ce83da5.png)
Grey placeholder text "J. Smith"

![image](https://user-images.githubusercontent.com/628064/196160907-ac611f84-b438-4fad-86b5-5f3ef54d0d95.png)
Entered green text "MR TM Martyn"

**Desktop (please complete the following information):**
 - OS: macOS Monterey 12.6
 - Browser: Chrome
 - Version: 106

**Additional context**
Using CCA contrast checker tool from TPGI
https://www.tpgi.com/color-contrast-checker/

**Comment by m1aw** [comment]
This issue has been fixed in versions [v5.20.0](https://github.com/Adyen/adyen-web/releases/tag/v5.20.0) and [v5.24.0](https://github.com/Adyen/adyen-web/releases/tag/v5.24.0) so please consider upgrading.
